def call(**kwargs):

    try:
        result = "Hello world"
    except Exception as exception:
        print (exception)
    
    return result